python/pyarrow/tests/test_parquet.py

```diff
@@ -366,6 +366,20 @@ def test_multithreaded_read():
 
 
 @parquet
+def test_min_chunksize():
+    data = pd.DataFrame([np.arange(4)], columns=['A', 'B', 'C', 'D'])
+    table = pa.Table.from_pandas(data.reset_index())
+
+    buf = io.BytesIO()
+    pq.write_table(table, buf, chunk_size=0)
```

Comment:
i think the issue was the user did len(foo) / K and it truncated to zero. We could have it raise an exception instead